azure/virtual-network/db-subnet.tf

```diff
@@ -0,0 +1,55 @@
+resource "azurerm_network_security_group" "db" {
+  name                = "${local.default_name}-db"
+  location            = var.region
+  resource_group_name = var.resource_group_name
+
+  security_rule {
+    name                       = "${local.default_name}-db"
+    priority                   = 100
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = 5432
+    destination_port_range     = 5432
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
+  }
+
+  tags = coalesce(var.tags, local.default_tags)
+}
+
+resource "azurerm_subnet" "db" {
+  name                 = "${local.default_name}-db-subnet"
```

Comment:
pls fix faulty merge, this file included fixes that haven't been merged correctly: https://github.com/dbl-works/terraform/pull/322/files#diff-e5a58c2e0c8d4b4f2a5f57fdd0b4d2c084b1b5482f9103cc8c898c9bfd9b7888L2

Reply:
@samkahchiin ping